Core/Core/Api/Operations/Operations.Serialize.cs

```diff
@@ -47,7 +46,7 @@ public static partial class Operations
   /// </summary>
   /// <param name="objects"></param>
   /// <returns></returns>
-  [Obsolete("Please use the Serialize(Base @object) function. This function will be removed in later versions.")]
+  [Obsolete("Please use the Serialize(Base @object) function. This function will be removed in later versions.", true)]
```

Comment:
Does this need to now be (Base value)  ?

Reply:
It should be, I'll fix in another PR